config/django.dev.env

```diff
@@ -0,0 +1,5 @@
+# Django
+DJANGO_SETTINGS_MODULE=meetupselector.settings.dev
+SECRET_KEY=askdfj98asdf0-asdyf7yhaasdfa87asdf723*
```

Comment:
Secrets should not be pushed into repo

Reply:
My suggestion is to create a single `.env` (example.env) file in project root folder, with all environment variables.
That `.env` file is automatically read by Docker when creating containers and is used **only for development**. Developers shall create their own `.env` file in localhost using `example.env` file as a template. 
So, `example.env` is sync with git and `.env` should be added to `.gitignore`.
Then, in CI, when deploying to production, all environment variables are created on the fly.